### ISSUE TITLE
FYI: Warning from browser on download - all is well!

### ISSUE THREAD

**Issue by xcasxcursex** [mirinsoft/sharpapp#28]
TL;DR nothing to worry about

Given that firefox uses Google's APIs for this,  I'm sure it'll come up in other browsers soon:

The download icon in the toolbar gets a yellow dot badge:
![image](https://user-images.githubusercontent.com/58130821/83877028-2a42e880-a77d-11ea-84b7-6d7664ec281c.png)
There is a prompt in the popup:
![image](https://user-images.githubusercontent.com/58130821/83877091-45155d00-a77d-11ea-8778-9d9df52d6f35.png)
Which changes on mouseover:
![image](https://user-images.githubusercontent.com/58130821/83877110-4fcff200-a77d-11ea-8c8f-656f719f2e4e.png)
If you click the item for more information you get this dialog:
![image](https://user-images.githubusercontent.com/58130821/83877193-70984780-a77d-11ea-956b-d04ed5b999c8.png)

Of course, the file is still safe:
![image](https://user-images.githubusercontent.com/58130821/83877262-93c2f700-a77d-11ea-9339-01db7143c28e.png)

PS: Scanned with the most recent AV signatures as of the time in the screenshot, firefox nightly latest version (79.0a1 2020-06-05), and the file in my screenshots is called *(3) because I downloaded it a few times, but it does this anyway.

PPS: Isn't it funny that I've literally never seen this before and the first time I see it, it's on something Microsoft would prefer to erase from existence......

**Comment by Belim** [comment]
I have partly adressed this here
http://www.mirinsoft.com/blog/19-apps/34-stable-release-of-new-spydish-app-is-out

I have to request a verification to Google from stopping Chrome's Safe browsing features (also used by Firefox) classifying my app as malicious. It has become really bad how you are controlled from all sides. But at least there was a reaction. 

You make efforts, develop and want to help and in the end no user trusts you, because some stupid heuristics tells you that the app is not safe. Nobody is turning on its brain.  It really makes no fun.

**Comment by xcasxcursex** [comment]
Spydish looks amazing!!!!! Well done!!!

Let us hope that people who are smart enough to use apps like spydish and sharpapp, are also smart enough to understand the false positives and that the apps are safe.

**Comment by Belim** [comment]
It's getting better. The higher the level of awareness, the less they can do. SharpAapp and Spydish are not flagged anymore by Google

**Comment by xcasxcursex** [comment]
I'm afraid I'm still getting the warning :( There are some delays between FF and Google ([in order to maintain privacy and performance, checks against the Safe Browsing database are performed periodically rather than every time](https://feeding.cloud.geek.nz/posts/how-safe-browsing-works-in-firefox/)) so I'm sure it'll come good soon. I'll let you know :)

**Comment by Belim** [comment]
I'm somewhat amazed as I've tested it on several Machines with latest Chrome/Firefox versions and Safe Broswing don't mess up

**Comment by xcasxcursex** [comment]
When I tried it at the time of that post, I got the warnings on both files. Trying it now, sharpapp is OK but spydish still warns me. I'm sure they will both be whitelisted very soon :) It's just the delay of my browser getting the hashes of the files from google via mozilla.

**Comment by xcasxcursex** [comment]
I'm now getting warnings on both downloads again. I guess they only whitelisted that one release. What a flawed system.

**Comment by Belim** [comment]
Google search console is not showing any security messages for my site and downloads. Cant do anything here actually.
I hope they will update their database again :weary: let us wait a day or so

**Comment by xcasxcursex** [comment]
Good news, I'm not getting warnings on either app now. Let's hope it stays that way :)
